graphrag_sdk/kg.py

```diff
@@ -149,16 +149,16 @@ def ask(
         )
 
         (context, cypher) = cypher_step.run(question)
-
```

Comment:
every chat interaction is accommodated with history, starting with an empty conversation and growing as the chat evolves. there shouldn't be two modes (with / without) history. there's only one mode and it includes history.